src/networks.json

```
@@ -1196,6 +1196,22 @@
     "start": 94085,
     "logo": "ipfs://QmaQxfwpXYTomUd24PMx5tKjosupXcm99z1jL1XLq9LWBS"
   },
+  "2611": {
+    "key": "2611",
+    "name": "Redlight Mainnet",
+    "shortName": "REDLC",
+    "chainId": 2611,
+    "network": "mainnet",
+    "multicall": "0x516C7A3F307eCb28F7F88AFfD31F26484c2e1BA2",
+    "rpc": [
+      "https://dataseed2.redlightscan.finance"
+    ],
+    "explorer": {
+      "url": "https://redlightscan.finance/"
+    },
+    "start": 6454240,
```

Should be same as multicall creation block :)
```suggestion
    "start": 6504228,
```